packages/core/src/components/ui/Dialog/Dialog.Modal.js

```diff
@@ -29,6 +29,7 @@ const Modal = forwardRef(({
       focusTrapOptions={{
         fallbackFocus: document.getElementById(DIALOG_CONTAINER_ID),
         initialFocus: titleRef.current,
+        allowOutsideClick: true,
```

Comment:
please add prop for this

Reply:
@bhoomij do we relay need that. Because all of our modal components need this option

Reply:
@dmytroshch  will this close the modal if clicked anywhere outside the modal on the page ?
I asked in that context thinking this will now always close modal if clicked outside the modal, while certain cases for ufx-ui need this feature that modal should only close when `close` button is clicked explicitly and not when clicking on view somewhere outside modal.